client/src/components/charts/LineChart.jsx

```diff
@@ -365,7 +364,7 @@ export default class LineChart extends Component {
 
                   <AxisLeft
                     scale={yScale}
-                    left={xScale(series.data[0].timestamp)}
```

Comment:
this is the first item in the data array - so it would mean that the axis is inline with the first node.